packages/common/src/store/cache/reducer.ts

```diff
@@ -87,7 +87,9 @@ const unwrapEntry = (entry: { metadata: any }) => {
 const forceUpdateKeys = new Set([
   'field_visibility',
   'followee_reposts',
-  'followee_saves'
+  'followee_saves',
+  'associated_wallets',
```

Comment:
what's this for?

Reply:
i refactored those two over to this one cause the intent is the same, which is to always prefer the incoming value